src/github.py

```diff
@@ -8,6 +8,30 @@ def __init__(self, group: str, repo: str, auth_token: str = None) -> None:
         self.repo: str = repo
         self.auth_token: str = auth_token
 
+    def get_test_result(test, logfiles, col):
+        test_exist = False
+        test_col = ':green_circle:'
+        for i in range(len(logfiles)):
+            if test in logfiles[i]:
+                test_exist = True
+                if col[i] == ':red_circle:' or col[i] == ':lock:' or col[
+                        i] == ':wastebasket:' or col[i] == ':hourglass:':
+                    test_col = ':red_circle:'
+                elif col[
+                        i] == ':yellow_circle:' and not test_col == ':red_circle:':
+                    test_col = ':yellow_circle:'
+        return (test_exist, test_col)
+
+    def add_test_to_text(test, test_exist, test_col, col, logfiles, text):
```

Comment:
This function creates a string and appends it to `text` and returns `text`. The content of `text` is irrelevant to this function. Have you considered changing this function, such that it just returns the appended string, and make the caller append it? How about extracting `if test_exists:` to the caller?